src/main/java/seedu/lifeasier/ui/Ui.java

```diff
@@ -208,4 +208,37 @@ public void showSelectWhichNoteMessage() {
         System.out.println("Please select the notes you want to view:\n");
     }
 
+    public void showFreeTimeMessage(int startHour, int endHour, int duration) {
+
+        if (duration != 0) {
+            System.out.println("You have " + duration + " hours of free time between " + startHour
+                    + ":00 and " + endHour + ":00!");
+            System.out.println("Hope you can schedule something in this time!");
+
+        } else {
+            System.out.println("Unfortunately you have no free time today!");
+            System.out.println("You might want to relax a little!");
+        }
+
+        printSeparator();
+    }
+
+    public void showSleepTimeMessage(int earliestSleepTime, int latestWakeTime, int duration) {
+
+        if (earliestSleepTime == 0 && latestWakeTime == 24) {
+            System.out.println("You have nothing on for today and tomorrow!");
+        } else {
+            System.out.println("You have nothing on from " + earliestSleepTime + ":00 today to " + latestWakeTime
+                    + ":00 tomorrow!");
+        }
+
+        if (duration < 9) {
+            System.out.println("You can sleep for up to " + duration + " hours!");
+        } else {
+            System.out.println("You can sleep for the recommended 8 hours or longer!");
+        }
```

Comment:
Should the logic for like determining which messages to print be done not in the Ui class? I'm not very sure also, want to know your opinion

Reply:
Good point, I'll change it and see how it looks!

Reply:
The reason why I left it like this for now was because I found it difficult to have descriptive meaningful method names if I were to split it up.